pyspectral/tests/unittest_helpers.py

```diff
@@ -28,3 +28,21 @@ def assertNumpyArraysEqual(self, other):
         raise AssertionError("Shapes don't match")
     if not np.allclose(self, other):
         raise AssertionError("Elements don't match!")
+
+
+class CustomScheduler(object):
```

Comment:
It's already present in test_blackbody... https://github.com/pytroll/pyspectral/blob/main/pyspectral/tests/test_blackbody.py#L39-L53

Reply:
The scheduler is renamed and the duplicate removed in https://github.com/pytroll/pyspectral/pull/206/commits/234ac9189e74a79ea1ebac44c66e39688e3f7e49